ports/libzmq/README.md

```diff
@@ -0,0 +1,107 @@
+**NOTE**: QNX ports are only supported from a Linux host operating system
+
+Use `$(nproc)` instead of `4` after `JLEVEL=` and `-j` if you want to use the maximum number of cores to build this project.
+32GB of RAM is recommended for using `JLEVEL=$(nproc)` or `-j$(nproc)`.
+
+# Compile the port for QNX in a Docker container
+
+Pre-requisite: Install Docker on Ubuntu https://docs.docker.com/engine/install/ubuntu/
+```bash
+# Create a workspace
+mkdir -p ~/qnx_workspace && cd ~/qnx_workspace
+git clone https://github.com/qnx-ports/build-files.git
+
+# Clone libzmq
+git clone https://github.com/zeromq/libzmq.git && cd libzmq
+
+# Checkout version 4.3.5
+git checkout v4.3.5
+
+# Build the Docker image and create a container
```

Comment:
Might be easier to remove the "on ubuntu host" step and just say this step is optional, as the process is the same otherwise.